cluster/manifests/coredns-local/daemonset-coredns.yaml

```diff
@@ -24,15 +24,6 @@ spec:
         version: v1.2.0
         component: cluster-dns
     spec:
-      affinity:
```

Comment:
why did this affinity exist? so that the old setup did not get scheduled on a single node?

Reply:
Yes, it was in place to control whether pods from the new setup or the old setup would land on a node depending on the node label `cluster-dns`.